components/EmailValidator.vue

```diff
@@ -6,18 +6,20 @@
       label-class="mt-0"
       :state="true"
     >
-      <Field
-        ref="email"
-        v-model="currentEmail"
-        :rules="validateEmail"
-        type="email"
-        name="email"
-        :class="'email form-control input-' + size + ' ' + inputClass"
-        :center="center"
-        autocomplete="username email"
-        :placeholder="'Email address ' + (required ? '' : '(Optional)')"
-      />
-      <ErrorMessage name="email" class="text-danger font-weight-bold" />
+      <VeeForm ref="form" as="">
```

Comment:
Are there any issues with nested `VeeForm`?  In `CommunityEventModal` we have `VForm` which is wrapping this.

I'm assuming `VForm` and `VeeForm` are the same - if so then we should use just `VeeForm` as it's clearer.

Reply:
We need Form wrapper from vee-validate because otherwise, it won't show <error-message> component. We can either implement it on our own and show/hide depending on validation result or rely on vee-validate, but we need <Form> wrapper.

I'll rename VForm to VeeFrom in other places as well.

Reply:
Does it introduce an actual `form` element into the DOM?  If it does then some uses of this component will result in a nested `form` element, which isn't valid HTML.

Reply:
No, it doesn't. I think having as="" makes it renderless

Reply:
Ok, great.  That's fine then.